report-zh.md

```diff
@@ -0,0 +1,248 @@
+
+这份报告的结论是，该计划的下一步应该是确定W3C工作组的职权范围。该工作组从相关的W3C、WHATWG和OGC（标准）工作组中获取建议和需求。
+
+<p class="info">会议视频、纪要以及参考材料均可在研讨会网站的<a href="{{ relBase }}/agenda">日程页面</a>找到。</p>
+
+<h2 id="introduction">简介</h2>
+
+加拿大自然资源部（NRCan）赞助了首个关于Web地图标准化的联合研讨会，该研讨会于2020年9月21日至10月2日在线上举行。研讨会由万维网联盟（W3C）和开放地理联盟（OGC）共同牵头举办，并由来自NRCan、W3C、OGC、加拿大环境与气候变化部（ECCC）、美国国家地理空间情报局（NGA）、美国国家航空和宇宙航行局（NASA）的成员，以及来自HTML地图社区的志愿者组成了组织委员会。该研讨会原计划与2020年6月的OGC技术委员会会议一同在线下举行，但由于2019新冠疫情（COVID-19），研讨会改为线上的形式，并对公众免费开放。
+
+本次研讨会通过[网站](https://www.w3.org/2020/maps/)发布了对与会者、论文以及讲座的征集，并通过W3C和OGC会员沟通渠道共同推广。研讨会是按主题划分为不同板块，所有主题都以Web平台中地图标准化的概念为中心。主题包括：可访问性、国际化、隐私和安全、性能、增强现实、Web地图标准和技术，以及利益相关者的观点。
+
+会议共有超过150人次[报名](https://www.w3.org/2020/maps/supporting-material)，共收到13份意见书、33份报告、7个专家组讨论和3个分组讨论会。所有的会议都进行了记录，每天的会议记录与当天的[议程项目](https://www.w3.org/2020/maps/agenda#day-1)相关联。会议鼓励大家对演讲和专家组讨论进行讨论（目前仍是），可以在会议中通过[实时对话](https://gitter.im/Maps4HTML/W3C-OGC-Workshop-Maps-For-Web)讨论，也可以在会后，在单独的[Web标准讨论论坛主题](https://discourse.wicg.io/c/web-mapping/22)中对每个演讲、专家组讨论或分组讨论进行讨论。为了方便回顾，每个单独的讨论主题都会链接到相应的会议回放。
+
+<h2 id="workshop-themes">研讨会主题</h2>
+
+<h3 id="accessibility-a11y">可访问性（无障碍访问）</h3>
+
+Nic Chan和Robert Linder所做的一项[分析](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Nic_Chan/index.html)，是根据Web内容无障碍指南（WCAG）来衡量现有的Web地图嵌入解决方案的可访问性。简而言之，他们所评估的Web地图框架中，没有一个符合WCAG 2.1的所有成功标准。而对于某一些标准，被评估的框架中则没有一个能够达到。出于对此次评估的回应，一些框架维护者改善了框架的可访问性。作为研讨会的初步结果，这让人很有信心。
+
+Nicolò Carpignoli和Joshue O 'Connor对[无障碍Web地图的要求](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Nicolo_Carpignoli-Maps-for-the-Web.pdf) 进行了概述。他们强调的一个重要需求是可访问的、基于文本的地理位置“注释”。根据标准的地理模型，这些文本注释对应于对OGC的SFS简单要素标准。这个OGC标准描述了简单几何对象的通用架构，并且已经被KML、GeoPackage和GeoJSON等几个流行标准所采用。地理注释的标准化需要找到一种方法，在HTML中实现这些特性。浏览器支持带注释的地理元素的隐式先决条件是浏览器对地图呈现的控制（用户代理需要知道在哪里呈现该要素）。因此，这些需求似乎含蓄地认可了将地图集成到HTML的理念和相关的无障碍标准（CSS、ARIA、WCAG）。报告还强调有必要将主要信息与背景信息或周围的“噪音”分开。也就是说，这是一种基于层的地图模型，可以让使用计算机辅助的用户关注相关层。
+
+Claudia Loitsch和Julian Striegl描述了如何使用地图来改善[室内位置的可访问性](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Claudia_Loitsch_Julian_Striegl.pdf) 。他们的研究表明，目前室内地图的覆盖率非常低。更重要的是，对于如何让地图和地理数据能够满足不同情景和不同用户需求，目前还没有统一的标准。他们总结，由于室内地图数据的市场预计在未来几年将显著增长，上述问题为制定此类标准提供了绝佳机会，以免在不兼容的解决方案上消耗甚至浪费绘图工作。
+
+物理无障碍是指为身体残疾者提供的设施，如轮椅坡道等。WheelMap.org网站的Sebastian Felix Zappe认为[HTML中的地图可以被用于“残障主流化”](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Sebastian_Felix_Zappe_Holger_Dieteric.pdf)，也就是说，确保无障碍信息通过标准的制定，成为所有Web地图服务的一部分。
+
+来自Smith-Ketttlewell眼科研究所的Brandon Biggs[强调了向盲人和视力受损用户展示地图内容的重要性](https://www.w3.org/2020/maps/supporting-material-uploads/position-statements/Brandon_Biggs-Smith-Kettlewell.pdf)。WCAG标准和技术目前还没有涵盖Web地图中图形信息的复杂性，也没有涵盖将这种空间信息传达给非可视Web用户的方法。随着浏览器新增的内置功能，比如Web振动、音频和语音识别API，可以为Web创建更高级的非可视化地图。为了生成数字或振感有声地图，地理空间数据集中的每个要素都需要名称和完整的几何特征。Biggs开发的非视觉地图使用了空间碰撞检测，用户操作他们在空间中的虚拟位置，当虚拟位置与要素发生碰撞时，就会读出要素的名称属性。现有的栅格地图和未标记的特征向量集不能提供所需的文本描述。即使是将要素标记为点标记的地图也一样，因为要素在空间中的比例是未知的。Biggs的结论是，如果不考虑非可视地图对Web地图要素信息的数据建模要求，构建标准HTML组件是毫无意义的。这指出了建立Web地图标准需要解决的至少两个关键问题。
```

Comment:
s/Smith-Ketttlewell/Smith-Kettlewell

Reply:
Sorry for the typos and now fixed. Thanks @Malvoz